dts/msm8916/msm8939-huawei-kiwi.dts

```diff
@@ -16,4 +16,7 @@
 
 	model = "Huawei Honor 5X";
 	compatible = "huawei,kiwi", "qcom,msm8939", "lk2nd,device";
+
+	// fix modem not connecting to cellular network
```

Comment:
I'd probably word it like this, but I'm not saying it strictly is better.

```suggestion
	// Modem doesn't come up without this for unknown reasons

Reply:
+1 for longer explanation but I'd rather like to see a link to downstream kernel sources/dts that lists this value (if such source tree is even available) or maybe an explanation how one could extract these from the decompiled dtb.

I'm not strictly opposed to linking information sources there

Reply:
How about something like this: https://github.com/LineageOS/android_kernel_huawei_kiwi/search?q=modem_id